x/pocketcore/types/service.go

```diff
@@ -130,7 +131,8 @@ func (r Relay) RequestHash() []byte {
 	}{r.Payload, r.Meta}
 	res, err := json.Marshal(relay)
 	if err != nil {
-		panic(fmt.Sprintf("cannot marshal relay request hash: %s", err.Error()))
+		fmt.Println(fmt.Errorf("cannot marshal relay request hash: %s", err.Error()))
+		os.Exit(1)
```

Comment:
```suggestion
  log.Fatalf("cannot marshal relay request hash: %s", err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)